map.css

```diff
@@ -0,0 +1,24 @@
+.game {
+  display: flex;
+  height: 33%;
+  margin-top:-2em;
```

Comment:
What's an em?
There are a handful of non-spaced  arguments, is that intentional?